public/docs/getting_started/quickstart.md

```diff
@@ -0,0 +1,337 @@
+# Quickstart
+
+This page will take you through the basics of creating a Truffle project and migrating (deploying) a smart contract to a blockchain.
```

Comment:
I think you should just use `deploying` here, since it's more colloquial, and then later describe that migrating/migrations are the mechanism for deployment.